ui/src/app/edge/history/abstracthistorytimeperiod.ts

```diff
@@ -0,0 +1,69 @@
+import { interval, Subject } from 'rxjs';
+import { JsonrpcResponseError } from 'src/app/shared/jsonrpc/base';
+import { QueryHistoricTimeseriesDataRequest } from 'src/app/shared/jsonrpc/request/queryHistoricTimeseriesDataRequest';
+import { QueryHistoricTimeseriesDataResponse } from 'src/app/shared/jsonrpc/response/queryHistoricTimeseriesDataResponse';
+import { Service, ChannelAddress, Edge, EdgeConfig } from 'src/app/shared/shared';
+import { takeUntil } from 'rxjs/operators';
+
+export abstract class AbstractHistoryTimePeriod {
+
+    protected refreshTimePeriod = interval(300000);
```

Comment:
A comment would be highly appreciated here. What does refreshTimePeriod do, why is it protected and not private and what does 300000 mean?